packages/forms-web-app/src/views/projects/documents.njk

```diff
@@ -1,26 +1,30 @@
 {% extends "layouts/project-3-columns.njk" %}
 
 {% set activeProjectLink = 'about-the-application' %}
-
-{% block columnTwo %}
+{% block resultSec %}
   <div class="govuk-grid-row">
     <h2 class="govuk-heading-l">Project application documents</h2>
-
+      <form action="/projects/{{ caseRef }}/documents/1" method="POST" novalidate>
```

Comment:
As per my route comment - this doesn't need to set the `method` attribute since we will use the default `GET` method.